app/src/main/res/drawable/google_g.xml

```diff
@@ -0,0 +1,15 @@
+<vector android:height="24dp" android:viewportHeight="46.0"
+    android:viewportWidth="46.0" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#4285f4"
```

Comment:
use attr.xml and styles.xml for all your color needs.
other number can be as they are